content/en/blog/2022-11-11-prove-the-authenticity-of-helm-charts/index.md

```diff
@@ -0,0 +1,276 @@
+---
+author: developer-guy
+date: 2022-11-15 12:30:00+00:00
+title: Verify the integrity of the Helm Charts stored in OCI-compliant registries as OCI artifacts
+description: "We'll talk about one of the newest support in Flux v0.36 that enables you to prove the authenticity of the Helm charts we manage through the HelmChart resources with the help of the cosign integration."
+url: /blog/2022/11/verify-the-integrity-of-the-helm-charts-stored-as-oci-artifacts-before-reconciling-them-with-flux
+tags: [oci, security]
+resources:
+- src: "**.{png}"
+  title: "Image #:counter"
+---
+
+The cosign integration was one of the most important features we shipped in the Flux [v0.35 release](https://github.com/fluxcd/flux2/releases/tag/v0.35.0). After that, we wrote a [blog post](/blog/2022/10/prove-the-authenticity-of-oci-artifacts/) that explains how to use the feature with [OCIRepository](https://fluxcd.io/flux/components/source/ocirepositories/) resources that enables fetching OCI artifacts from container registries, and if you haven't read it yet, we highly encourage you to go and check it out first.
+
+{{< imgproc verify-the-integrity-of-the-helm-charts-stored-as-oci-artifacts-before-reconciling-them-with-flux Resize x700 >}}
```

Comment:
If you rename the file to "...-featured.png" it will turn up as the preview image for the blog post.